source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -79,6 +79,7 @@ public class CreateReleaseCommand : DeploymentCommandBase, ISupportFormattedOutp
 
         protected override async Task ValidateParameters()
         {
+            if (VersionNumber.Contains(' ')) throw new CommandException($"Release version '{VersionNumber}' is invalid, version cannot contain whitespace.");
```

Comment:
Rather than strip the whitespace for the user, tell them that they've entered the release version incorrectly as Server will strip the space when creating the release so it'd not match what the user passed through.